### DISCUSSION TITLE
Have you guys thoght of adding the functionality of cakebrew to Applite?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [milanvarady/Applite#36] [Q&A] [answered]
[Cakebrew](https://github.com/brunophilipe/Cakebrew) is a GUI for managing homebrew packages, but it looks like it's been abandoned since it hasn't been updated since 2021. It shows a lot of information like this:
![68747470733a2f2f7777772e63616b65627265772e636f6d2f6173736574732f696d672f6170702d62672e706e67](https://github.com/milanvarady/Applite/assets/42646062/2811fa99-ce08-466d-9065-33071e0bb3a3)

**Comment by milanvarady** [comment, marked as answer]
I'm not considering adding support for downloading formulae from the app because Applite was designed for non-technical users, so simplicity is key. Adding formulae would clutter up the interface and confuse users who don't know much about Homebrew. Also, formulae can only be used through the terminal anyway, so people who use them should know how to use the terminal.

**Reply by milanvarady** [reply]
But if you miss some functionality other than downloading formulae that Cakebrew had, feel free to suggest it.
